guides/ci/travis.md

```diff
@@ -0,0 +1,364 @@
+# Travis CI
+
+If you are new to continuous integration (or you want to refresh the basic concepts) we may start reading the [core concepts guide](https://docs.travis-ci.com/user/for-beginners/) and then a [short tutorial](https://docs.travis-ci.com/user/tutorial/).
+
+Now let’s see an example of using Travis CI with Crystal.
+
+## Build and run specs
+
+### Using `latest` and `nightly`
+
+When [creating a Crystal project](https://crystal-lang.org/reference/using_the_compiler/#creating-a-crystal-project) using `crystal init`, Crystal creates a `.travis.yml` file for us:
+
+```yaml
+# .travis.yml
+language: crystal
+
+crystal:
+  - latest
+  - nightly
+
+script:
+  - crystal spec
+  - crystal tool format --check
+```
+
+Although a basic configuration file, it will let’s use Travis CI. Now, let’s go to Travis CI dashboard to [add the GitHub repository](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci) From this moment, Travis CI will run the `specs` against the Crystal compiler (using both `latest` and `nightly` releases)
```

Comment:
```suggestion
Although a basic configuration file, it will let’s use Travis CI. Now, let’s go to Travis CI dashboard to [add the GitHub repository](https://docs.travis-ci.com/user/tutorial/#to-get-started-with-travis-ci). From this moment, Travis CI will run the `specs` against the Crystal compiler (using both `latest` and `nightly` releases)
```